microprofile/microprofile-rest-client-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/rest/client/v13/sslsupport/WrongKeystoreCDIClient.java

```diff
@@ -0,0 +1,13 @@
+package org.wildfly.swarm.ts.microprofile.rest.client.v13.sslsupport;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterRestClient(configKey = "WrongKeystoreCDIClient")
+@Path("resourceCDI")
```

Comment:
Please move `@Path` in all client interfaces to the method. And use the existing `SimpleResource` (which is at `/rest/simple`), as mentioned above.